splinterd/src/node/runnable/mod.rs

```diff
@@ -41,6 +42,30 @@ pub(super) enum RunnableNodeRestApiVariant {
     ActixWeb3(RunnableRestApi),
 }
 
+impl RunnableNodeRestApiVariant {
+    #[cfg(feature = "biome-credentials")]
+    pub fn with_biome_auth(self, provider: BiomeCredentialsRestResourceProvider) -> Self {
```

Comment:
the `auth_configs` value that these methods set in the `RestApiBuilder` will always get over written lower down in this file, line 117-118 `let (rest_api_shutdown_handle, rest_api_join_handle) = rest_api.with_auth_configs(auth_configs)` so I think you'll probably just need to update that chunk to do `append_auth` as well rather than `with_auth_configs`